README.md

```diff
@@ -21,14 +21,14 @@ For Kubernetes clusters:
 | **Components** | **Version** |
 | -------------- | ----------- |
 | Kubernetes     | 1.16+       |
```

Comment:
I think we would have to update kubernetes and openshift versions too based AKO requirements.

Reply:
I have updated the versions based on the documents https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/release-1.6.1/buildsettings.json and https://avinetworks.com/docs/ako/1.6/ako-compatibility-guide.